myuw/static/js/card/summary/schedule.js

```diff
@@ -95,7 +96,7 @@ var SummaryScheduleCard = {
                 case "B-term":
                     b_term.push(instructed_course_data.sections[i]);
                     break;
-                case "Full-term":
+                default:
```

Comment:
What is the rationale for this change? Do PCE/EOS courses not always have a summer term like SDB ones do?

Reply:
Yes, this is the fix for MUWM-4318  (EOS summer section has summer term value "".)